### PR TITLE
[WalletConnect] Expose eth_sendTransaction RPC

### PR DESCRIPTION
**Pull request by AlexBHarley**
### Description

To be compatible with ContractKit, our WalletConnect implementation exposes the `eth_signTransaction` RPC. However usually, clients would prefer to rely on the `eth_sendTransaction` RPC where the wallet (ie. Valora) also handles sending the transaction.

### Other changes

Fixed the RPC error that Valora returns when denying a request.

### Tested

Using the script 👇. This script won't be merged as we'll be taking a better approach to accessing this RPC method in ContractKit. However for the purposes of testing it's good enough.

### How others should test

Use [this](https://gist.github.com/AlexBHarley/6bac73e802f19a61ae4ba92de45e4ee2) script instead of `packages/sdk/wallets/wallet-walletconnect/scripts/run-in-memory-client.ts` in the monorepo. 

### Related issues

- Fixes https://github.com/celo-org/celo-monorepo/issues/8159

### Backwards compatibility

N/A. New method being added.

### CODE REVIEW

**Review by jeanregisser — APPROVED** [review]
LGTM 👍 

We'll have to think about an e2e test for WalletConnect to ensure it continues working as expected in the future.

File: packages/mobile/src/walletConnect/saga.ts [hunk 1]
```diff
@@ -148,6 +152,24 @@ export function* acceptRequest({
         case SupportedActions.personal_decrypt:
           result = yield call(wallet.decrypt.bind(wallet), account, Buffer.from(params[1]))
           break
+        case SupportedActions.eth_sendTransaction:
+          const kit: ContractKit = yield call(getContractKit)
+          const normalizer = new TxParamsNormalizer(kit.connection)
+          const tx: CeloTx = yield call(normalizer.populate.bind(normalizer), params)
+
+          const sendTxMethod = function* (nonce?: number) {
+            const txResult: TransactionResult = yield call(kit.connection.sendTransaction, {
```
Comment (jeanregisser) [inline packages/mobile/src/walletConnect/saga.ts:161]: no binding needed here?

Comment (AlexBHarley) [reply packages/mobile/src/walletConnect/saga.ts:161]: Nope. No idea why but it works

**Comment by Celoqa** [comment]
Hey @MuckT I have verified this issue using Android internal build V1.16.1 & Test Flight build V1.16.1(67) & observed following:
Attachments:



CC: @AlexBHarley

**Comment by Celoqa** [comment]
Hey @MuckT I have verified this issue using Android internal build V1.16.1 & Test Flight build V1.16.1(67) & observed following observations:

- **For Android:**

- [ ] We have observed blank screen when user select Wallet connect option from options [Attachment](https://drive.google.com/file/d/1ysLGI0NjlJwr0FiH6qliino3LwbkVUXD/view?usp=sharing)



- **For iOS:** 

- [ ] We have observed that nothing happens when user taps on "Connect withWalletConnect" button. [Attachment](https://drive.google.com/file/d/1EpR6oRj814wW46emEw59KSCllrbs3AsW/view?usp=sharing)

FYI: We have raised query ticket #871 for same
- **For Web:**

- [ ] We are able to connect wallet & able to observe error for all three (Test sendTransaction, Test signTypedData, Test signPersonal) . [Attachment](https://drive.google.com/file/d/1hQnatjPKDoh73l7W50qaDYgfh_cutcwP/view?usp=sharing)

**Verified On Devices & browsers:** Redmi Note 8 (9.0),iPhone SE (14.2), Firefox 89.0.2
CC: @AlexBHarley 
Thanks..!!

**Pull request merged**